mkt/extensions/tests/test_views.py

```diff
@@ -148,6 +149,11 @@ def setUp(self):
         self.list_url = reverse('api-v2:extension-list')
         self.user = UserProfile.objects.get(pk=2519)
 
+    def tearDown(self):
+        super(TestExtensionViewSetPost, self).tearDown()
+        # Explicitely delete the Extensions to clean up leftover files.
```

Comment:
Yeah, sorry, this is a drive-by fix. I stumbled upon it when re-launching the tests locally a bunch of times when writing this PR, but it was caused by earlier commits.